config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1580,3 +1580,52 @@ presubmits:
           limits:
             cpu: 4
             memory: 6Gi
+
+  - name: pull-kubernetes-e2e-inplace-pod-resize-containerd-main
+    cluster: k8s-infra-prow-build
+    always_run: false
+    optional: true
+    skip_report: false
+    run_if_changed: '^(pkg\/kubelet\/kuberuntime\/|test\/e2e\/node/)'
+    skip_branches:
+      - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-dashboards: sig-node-presubmits
+      testgrid-tab-name: pr-kubelet-gce-cluster-e2e-inplace-pod-resize-containerd-main
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221107-33c989e684-master
+        args:
+          - --repo=github.com/containerd/containerd=main
+          - --root=/go/src
+          - "--job=$(JOB_NAME)"
+          - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+          - "--service-account=/etc/service-account/service-account.json"
+          - "--upload=gs://kubernetes-jenkins/pr-logs"
+          - "--timeout=180"
+          - --scenario=kubernetes_e2e
+          - -- # end bootstrap args, scenario args below
+          - --check-leaked-resources
+          - --cluster=
+          - --env=ENABLE_AUTH_PROVIDER_GCP=true
```

Comment:
remove this, this is not needed (it's only needed for the `kubelet-credential-provider` job)

Reply:
fixed both issues. Please give it an ok-to-test to check if CI likes it(local make verify looks good).